tests/pio_tests/scenarios/basic_app_usecases.py

```diff
@@ -61,17 +61,13 @@ class BasicAppUsecases(BaseTestCase):
   def setUp(self):
     random.seed(3)
     self.log.info("Setting up the engine")
-
-    template_path = pjoin(
-        self.test_context.engine_directory, "recommendation-engine")
+    template_path = "https://github.com/chanlee514/template-scala-parallel-recommendation"
```

Comment:
They're temporary forks with org namespace updates (io.prediction >> org.apache.predictionio) to make the tests work. Maybe I can create a PR to each of the official template repos following v0.10 release, and then update this?